src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -60,6 +58,10 @@ internal abstract partial class AbstractNavigateToSearchService
             Document? searchDocument, string pattern, IImmutableSet<string> kinds,
             Func<RoslynNavigateToItem, Task> onResultFound, CancellationToken cancellationToken)
         {
+            // We're doing a real search over the fully loaded solution now.  No need to hold onto the cached map
+            // of potentially stale indices.
+            ClearCachedData();
+
```

Comment:
Why do we clear the cache twice?

Reply:
i'm not sure what this means.

Reply:
disregard, looks like a rendering issue in the VS Code web editor:
![image](https://user-images.githubusercontent.com/9797472/139960823-ee23f9f5-f127-4a1a-a94d-8d9a11c2dd53.png)